### ISSUE TITLE
[TF2] [Feature Request] Add a check to prevent players from getting stuck in Pumpkin Bombs

### ISSUE THREAD

**Issue by KimmyTF2** [ValveSoftware/Source-1-Games#6192]
Currently players can get stuck in a Pumpkin Bomb's location when it spawns. A check delaying a Pumpkin Bomb respawn as long as someone (or something) is standing where it spawns would be appreciated.

**Comment by Platina6978** [comment]
The same is true for other explodables like the ones found on 2fort_invasion. Those should also have a check added to them in case the above suggestion gets added.

**Comment by Flpstrike** [comment]
Good idea, when that used to happen... guess who exploded the pumpkin for me? the enemy or someone from my team?
